include/engine/plugins/tile.hpp

```diff
@@ -23,8 +25,13 @@ namespace plugins
 
 class TilePlugin final : public BasePlugin
 {
+  private:
+    routing_algorithms::BasicRoutingInterface<
+        datafacade::BaseDataFacade,
+        routing_algorithms::ShortestPathRouting<datafacade::BaseDataFacade>> routing_base;
+
   public:
-    TilePlugin(datafacade::BaseDataFacade &facade) : BasePlugin(facade) {}
+    TilePlugin(datafacade::BaseDataFacade &facade) : BasePlugin(facade), routing_base(&facade) {}
```

Comment:
access to `routing_base` seems like the wrong approach to me. The plugin should not handle routing-related questions.
As discussed later, I feel that using the normal unpack routine and getting access to edge-data via the normal graph and locating the respective edge should be the way to go.